ql/src/java/org/apache/hadoop/hive/ql/plan/Statistics.java

```diff
@@ -53,19 +53,21 @@ public State merge(State otherState) {
   private long numRows;
   private long runTimeNumRows;
   private long dataSize;
+  private long totalFileSize;
```

Comment:
Could you add a comment on what dataSize and totalFileSize mean? It will prevent us from getting confused when we check the code back after some time :)

Reply:
Done